### PR TITLE
New homepage

### PR DESCRIPTION
**Pull request by sbastn**
This PR moves the previous homepage to "Getting started" and replaces the content with quick access to specific API languages and feature announcements related to https://github.com/dnsimple/dnsimple-marketing/issues/544.

## QA 
Use https://deploy-preview-429--dnsimple-developer.netlify.app/ to browse around and make sure if UTM codes are correct for external links to the app and blog.

Screenshot 2022-10-04 at 13-28-16 Screenshot

### CODE REVIEW

File: content/index.html [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+<div class="pv4">
+  <h2 class="ma0 pb2 mb2 bb bw1 divider">New Features</h2>
+  <a href="https://blog.dnsimple.com/2022/06/support-webauthn-api/?utm_source=developer&utm_medium=web&utm_content=new-features" class="link dim" target="_blank">
+    <div class="flex flex-wrap justify-between items-center">
+      <div class="w-100 w-50-ns">
+        <img src="/assets/images/features/two-step-verification.png" alt="Security keys" class="br3">
+      </div>
+      <div class="w-100 w-50-ns pa3-ns">
+        <h3>Get Stronger Account Protection with Web Authentication (WebAuthn) Support</h3>
```
Comment (dallasread) [inline content/index.html:90]: We need to pull these directly from the blog. 😁

**Review by dallasread — APPROVED** [review]
Love it!

I'm running into a small issue with the blog article formatting:

image

It looks like `content/assets/images/vendors/linux-foundation-light.svg` is unused.

File: content/index.html (hunk 1 above)
Comment (sbastn) [reply content/index.html:90]: Yes, automation would be the next step. Right now, it's just an experiment. We'll be measuring if this generates any traffic in the next couple of months.

Comment (sbastn) [reply content/index.html:90]: I fixed the unused image and the spacing around the blog posts. Thank you!

**Comment by laetitiaeden** [comment]
@sbastn 

-  I see one typo at  "articles" missing l. Make sure to ping @itsalyse even if it after publishing the new page, she can double check the copy.


@sbastn and @dallasread 

- Based on Google Analytics, we see that top 3 of landing page being visited is Sandbox, I would like to mention that I believe we miss an opportunity for the developer that is looking at the page to try our service at no risks with the Sandbox. What do you think about having the sandbox stand alone (Option 1) or the developer tips section added to the page to showcase the sandbox environment (Option 2)
Option 1
image

Option 2
image

**Comment by sbastn** [comment]
Thanks @laetitiaeden for the feedback. I've added a section between the clients and getting started with sandbox information. This new section links to https://developer.dnsimple.com/sandbox/ since we list the credit card information to create a subscription

Screenshot 2022-10-04 at 16 47 36

**Comment by sbastn** [comment]
@laetitiaeden and I agreed to not install hotjar on the dev site for the time being and that the links with UTM codes should be good enough to measure interest.

**Pull request merged**